pubspec.yaml

```diff
@@ -1,10 +1,9 @@
 name: mime
 version: 1.0.2
```

Comment:
```suggestion
version: 1.0.2-dev
```

Reply:
Ah, `1.0.2` isn't published yet.

I think I had forgotten that I was going to publish it, so this looks good as is.